kroxylicious/src/main/java/io/kroxylicious/proxy/internal/DefaultFilterContext.java

```diff
@@ -18,17 +27,25 @@
  */
 class DefaultFilterContext implements KrpcFilterContext, AutoCloseable {
```

Comment:
The coupling between `DefaultFilterContext` and the `FilterHandler` is quite strong, so I'm minded to make it an inner class.